hera_sim/simulate.py

```diff
@@ -892,6 +893,9 @@ def _initialize_data(
                 "Otherwise, keywords must be provided to build a UVData object."
             )
 
+        if not self.data.future_array_shapes:
```

Comment:
I think we can add `# pragma: nocover` here as well.

Reply:
👍🏻